x/leverage/keeper/interest.go

```diff
@@ -101,6 +101,10 @@ func (k Keeper) AccrueAllInterest(ctx sdk.Context) error {
 		if err := k.SetBorrowAPY(ctx, coin.Denom, derivedRate); err != nil {
 			return err
 		}
+
+		if err := k.SetLendAPY(ctx, coin.Denom, derivedRate.Mul(utilization)); err != nil {
```

Comment:
@toteki can you remind me what `utilization` is here again?

nit: I think we should call the `derivedRate` variable `borrowRate` instead.

Reply:
Borrow utilization is an `sdk.Dec` ranging from 0 to 1, which represents the portion of a given lent asset that has been borrowed.

For example if lenders have lent a total of 22000 uatom (excluding reserves) and borrowers have borrowed 11000 (after interest), then uatom is at 50% borrow utilization.